docs/rpc/rpc-spans.md

```diff
@@ -108,16 +108,6 @@ different processes could be listening on TCP port 12345 and UDP port 12345.
 **[5]:** May contain server IP address, DNS name, or local socket name. When host component is an IP address, instrumentations SHOULD NOT do a reverse proxy lookup to obtain DNS name and SHOULD set `server.address` to the IP address provided in the host component.
 
 **[6]:** When observed from the client side, and when communicating through an intermediary, `server.port` SHOULD represent the server port behind any intermediaries (e.g. proxies) if it's available.
-
-`rpc.system` has the following list of well-known values. If one of them applies, then the respective value MUST be used, otherwise a custom value MAY be used.
```

Comment:
I think we need to use "full" to render the enum values for `rpc.system` here

Reply:
Fixed.